assets/shared/helpers/player/index.js

```diff
@@ -162,4 +167,21 @@ class Player {
 	}
 }
 
+/**
+ * Hook to get the video duration.
+ *
+ * @param {Object} player Player instance.
+ *
+ * @return {number} The video duration.
```

Comment:
I think `number | undefined` would be more appropriate here, no?

```suggestion
 * @return {number|undefined} The video duration.
```